tests/main/abort/task.yaml

```diff
@@ -1,5 +1,8 @@
 summary: Check change abort
 
+# Disabled for Fedora until test case is properly ported
+systems: [-fedora-*]
```

Comment:
I think this is not needed given that the system has been disabled at the `main` suite level in https://github.com/snapcore/snapd/pull/3365/files#diff-3c11e56e5f7f82b0f352d0fe1851a107R306. There are lots of `systems` entries for tasks in `main`, not sure if they or the entry at the suite level should be removed, but we shouldn't have both.